### PR TITLE
Get ready for local developement

### PR DESCRIPTION
**Pull request by sabinem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,116 +1,248 @@
...
+```
+
+If you want to be able to build the handbook locally from its 
+**Source**, then you need to set up the virtual python3 environment and install the 
+requirements, as shown below:
+
+```
+python3 -m venv p3venv 
 source p3venv/bin/activate
+(p3venv) pip install wheel
```
Comment (bellisk) [inline README.md:75]: I wonder if including `(p3env)` at the start of these lines will confuse users, because they might try to copy that into the terminal as well. What do you think?

File: README.md [hunk 2]
```diff
@@ -1,116 +1,248 @@
...
 (p3venv) pip install -r requirements.txt
-(p3venv) cp .env.dist .env
```
Comment (bellisk) [inline README.md:32]: Why did you remove this line?

File: README.md [hunk 3]
```diff
@@ -1,116 +1,248 @@
...
 ```
 
-## Build the German Documentation
+Now everything is setup and you will be able to build the handbook locally and then view it 
+in your browser.
 
-The German documentation can be built locally like this:
+## Build the German locale
+
+The German locale is build from `source/locale/de` with the following command:
```
Comment (bellisk) [inline README.md:84]:
```suggestion
The German locale is built from `source/locale/de` with the following command:
```

File: README.md [hunk 4]
```diff
@@ -1,116 +1,248 @@
...
-
 - go to `build/de/` 
 - run `index.html` in a browser of your choice
 
-## Update French translations
 
-If you need on the French translations always start with a clean master repo:
+## Build the French locale
+
+The French locale is build from `source/locale/fr` with the following command:
```
Comment (bellisk) [inline README.md:96]:
```suggestion
The French locale is built from `source/locale/fr` with the following command:
```

File: README.md [hunk 5]
```diff
@@ -1,116 +1,248 @@
...
+In order to make changes just edit the corresponding `.rst` files.
+
+:warning: be aware that the French translations will stop working if you change German
+text, that has a French translation stored in the French locale.
+
+## Update the French Translation
+
+When you changed a text in the **Source**, that has a translation in the French Locale,
+you also need to change the corresponding extracted string in the Locale, so that the new 
+string can be found together with it's' translation when the Locale is build:
```
Comment (bellisk) [inline README.md:121]:
```suggestion
string can be found together with its translation when the Locale is built:
```

File: README.md [hunk 6]
```diff
@@ -1,116 +1,248 @@
...
+```
+
+:WARNING: Use this command with care as it has the following pitfalls:
+
+- some translations are marked with `#, fuzzy` in the Locales by the command and this command has to be removed, so that the translations will work
+- The command can't be restricted to just some files: it will update all file in the Locales that it identifies as needing a change
+
+## Linkchecker
+
+Sphinx also comes with a buildin linkchecker command.
```
Comment (bellisk) [inline README.md:235]:
```suggestion
Sphinx also comes with a builtin linkchecker command. 
```

File: README.md [hunk 7]
```diff
@@ -1,116 +1,248 @@
...
+in your browser.
 
-The German documentation can be built locally like this:
+## Build the German locale
+
+The German locale is build from `source/locale/de` with the following command:
 
 ```
-(p3venv) cd docs
-(p3venv) make clean
```
Comment (bellisk) [inline README.md:41]: Why did you remove this line? If you don't think it's necessary here, I think it would still be nice to include it somewhere, with a note saying what `make clean` does (removes the `build/` directory).

File: README.md [hunk 8]
```diff
@@ -1,116 +1,248 @@
...
 (p3venv) make -e SPHINXOPTS="-D language='de'" BUILDDIR="build/de" html
```
Comment (bellisk) [inline README.md:87]: I think it could be useful to say which directory they have to run this command from. Otherwise, "The German locale is built from `source/locale/de`" could make them think they have to navigate to `source/locale/de` before running it. (The same for building the French locale below.)

File: README.md [hunk 9]
```diff
@@ -1,116 +1,248 @@
...
+#: ../../source/index.rst:15
+msgid ""
+"Gut, dass Sie Open Government Data publizieren möchten! Wir "
+"unterstützen Sie dabei."
+msgstr ""
+"Vous souhaitez publier des données publiques ouvertes? Nous vous "
+"apportons notre soutien!"
+```
+
+Once that is done the French locale can be build again and the translation will be fixed.
```
Comment (bellisk) [inline README.md:171]:
```suggestion
Once that is done the French locale can be built again and the translation will be fixed.
```

File: README.md [hunk 10]
```diff
@@ -1,116 +1,248 @@
...
+
+## Extracting translatable strings from the Source
+
+The `.po` files in the Locales don't have to be set manually: they can be generated.  
+The way to generate them is the following:
+
+### Command for translatable strings extraction:
+
+The following comman extracts all translatable strings from the **Source**:
```
Comment (bellisk) [inline README.md:180]:
```suggestion
The following command extracts all translatable strings from the **Source**:
```

File: README.md [hunk 11]
```diff
@@ -1,116 +1,248 @@
...
 
 ```
 make gettext
 ```
 
-This will generate files located in `build/gettext/` with translatable 
-strings extracted from the `.rst` files. 
-Now generate the translation files with the following command:
+It generates a new directory in the `build` directory: `build/getext`
```
Comment (bellisk) [inline README.md:186]:
```suggestion
It generates a new directory in the `build` directory: `build/gettext` 
```

File: README.md [hunk 12]
```diff
@@ -1,116 +1,248 @@
...
-Now generate the translation files with the following command:
+It generates a new directory in the `build` directory: `build/getext` 
+The directory tree matches that of the `source/locale` directory trees.
+So for every `.rst` file in the **Source** it contains a `.pot` file 
+
+### Identify the translatable String for your Source change
+
+Find the `.pot` file that contains the translatable strings. 
+
+In the example above this would be `build/getext/index.pot`
```
Comment (bellisk) [inline README.md:194]:
```suggestion
In the example above this would be `build/gettext/index.pot`
```

File: README.md [hunk 13]
```diff
@@ -1,116 +1,248 @@
...
-The generated files will be located in `source/locale/fr/LC_MESSAGES` and 
-`source/locale/de/LC_MESSAGES`. Only the files in 
-`source/locale/fr/LC_MESSAGES` need further work: 
-Fill in the translations in the `.po` files. Once you are ready commit your 
-updated `.po` files to github.
+### Transfer the translatable string to the Locale
 
-To test the translations do the following to
-compile `.mo` files and build the translated html files for both languages:
+Copy thins into the `locale/fr/LC_MESSAGES/index.po` and replace the old phrase there by this
```
Comment (bellisk) [inline README.md:204]:
```suggestion
Copy this into the `locale/fr/LC_MESSAGES/index.po` and replace the old phrase there with this 
```

File: README.md [hunk 14]
```diff
@@ -1,116 +1,248 @@
...
+Locales. 
 
-You can run it with make:
+```
+sphinx-intl update -p build/gettext -l de -l fr
+```
+
+:WARNING: Use this command with care as it has the following pitfalls:
+
+- some translations are marked with `#, fuzzy` in the Locales by the command and this command has to be removed, so that the translations will work
```
Comment (bellisk) [inline README.md:230]:
```suggestion
- some translations are marked with `#, fuzzy` in the Locales by the command and this line has to be removed, so that the translations will work
```

File: README.md (hunk 1 above)
Comment (sabinem) [reply README.md:75]: I wanted to make clear that the virtual python environment needs to be activated. But may be you are right: it might be more confusing: so I will remove it.

**Review by bellisk — APPROVED** [review]
